dom0/sd-app-files.sls

```diff
@@ -18,3 +18,18 @@ install-securedrop-client-package:
       - securedrop-client
     - require:
       - sls: fpf-apt-test-repo
+
+install-securedrop-log-package:
+  pkg.installed:
+    - pkgs:
+      - securedrop-log
```

Comment:
It might be easier to read by adding this as a list item to the previous task

Reply:
Definitely fold into lists where possible. In salt, all individual tasks _must_ be uniquely named, whereas in this PR the `install-securedrop-log-package` task name is duplicated in several locations. If that's not throwing an error, we probably got lucky in that the states aren't combined for a single run... still, needs to be cleaned up.